ingest.py

```diff
@@ -1,316 +0,0 @@
-import re
-from functools import partial
-from itertools import izip
-from more_itertools import unique_everseen
-from urlparse import urlparse
-
-import arrow
-import requests
-from bs4 import BeautifulSoup
-from funcy import notnone, walk_keys, without
-
-from app import db
-from utils import find_player_name, multiple_replace
-
-
-_WINLOSS_REGEX = re.compile('.*?\((.*?)\)')
-
-
-class Ingester(object):
-    @staticmethod
-    def _date_conversion(text):
-        """
-        Converts text to a date.
-
-        :param text: String to convert.
-        """
-        return arrow.get(text).datetime
-
-    @staticmethod
-    def _home_conversion(text):
-        """
-        Converts text to a boolean stating whether a game was a home game.
-
-        :param text: String to convert, one of "" or "@".
-        """
-        return text != '@'
-
-    @staticmethod
```

Comment:
I get the idea of static methods here, but why have them when you're mostly calling `self.$static_method_name`? (Serious question)